### DISCUSSION TITLE
Custom keys to go up/down to select from a list of "commands/URLs/etc"

### DISCUSSION THREAD

**Discussion by mohaseeb** [qutebrowser/qutebrowser#6938] [Q&A] [answered]
Hi,

In the commands list (e.g. after hitting `:`) or URL list (e.g. after hitting `o`) one can use the the `up`/`down` arrow keys to go up/down these lists (see the example below). I'd like to reuse `ctrl-k`/`ctrl-j` instead. Is there a way to rebind up/down "commands?" to `ctrl-k`/`ctrl-j`?

**Example of what I'm after**
"I want to use `ctrl-j` to move the cursor down to "tab-focus" instead of hitting the `down` arrow key"
![](https://www.qutebrowser.org/doc/img/completion.png)

Thanks!

**Comment by The-Compiler** [comment, marked as answer]
qutebrowser has different modes where keys are bound in, like vim. All of its keybindings can be customized, as everything follows a "keys bound to qutebrowser commands" model.

The completion is shown in "command" mode. Thus, you can use `:bind --mode command <Up>` and the same with `<Down>` to view the commands bound to those keys - they are `completion-item-focus --history prev` and `completion-item-focus --history next` (remove the `--history` if you'd prefer to not go through command history if no text was added after the `:`).

Finally, to bind new keys to that, use something like `:bind --mode command <Ctrl-j> completion-item-focus --history next`.

**Reply by mohaseeb** [reply]
Thanks @The-Compiler for the thorough explaination; it works as hoped :)
